turtlebot3: #namespace
    task_and_robot_environment_name: 'TurtleBot3World-v0'
    ros_ws_abspath: "/home/adrian/trex_ros_ws"
    running_step: 0.04 # amount of time the control will be executed
    pos_step: 0.016     # increment in position for each command
    
    #qlearn parameters
    alpha: 0.1
    gamma: 0.7
    epsilon: 0.9
    epsilon_discount: 0.999
    nepisodes: 500
    n_checkpoints: 5 # Number of checkpointed models to be saved
    nsteps: 10000
    running_step: 0.06 # Time for each step in a training episode
    total_timesteps: 10000 # Total number of timesteps in an episode
    checkpoint_dir: learner/ppo2_checkpoints/turtlebot3 # Relative path to save checkpointed models
    pref_model_dir: learner/pref_model/trex_reward_turtlebot3